src/events/pollVotationResultsUpdate.js

```diff
@@ -0,0 +1,150 @@
+const { Events, roleMention, userMention } = require('discord.js');
+const { VOTE_POINTS } = require('../config');
+const { translateLanguage } = require('../languages');
+
+const tagIds = VOTE_POINTS.TAG_IDS;
+
+const getPointType = (questionPart) => {
+  const isBoostedPoint = questionPart.includes('boosted');
+  return isBoostedPoint ? tagIds.boostedPointTagId : tagIds.addPointTagId;
+};
+
+const parseQuestionInput = (questionField) => {
+  const questionText = questionField?.text;
+
+  if (!questionText) {
+    throw new Error(translateLanguage('votePoints.notFoundPollDescription'));
+  }
+
+  const parts = (questionText || '').split('|').map((part) => part.trim());
+
+  if (parts.length < 2) {
+    throw new Error(translateLanguage('votePoints.notFoundDescriptionParts'));
+  }
+
+  return {
+    pointType: parts[0].toLowerCase(),
+    userId: parts[1],
+  };
+};
+
+/**
+ * Retrieves the element(s) with the highest 'voteCount' from a question answers collection.
+ * In case of a tie, it returns all elements that share the maximum count.
+ *
+ * @param {Map<any, { id: any, text: string, voteCount: number }>} pollAnswers -
+ * The collection of PollAnswer objects (or similar) to analyze. It is expected that the
+ * values of the collection are objects with the properties 'id', 'text', and 'voteCount'.
+ * @returns {Array<{ id: any, text: string, voteCount: number }>} -
+ * An array containing the object(s) with the highest 'voteCount'.
+ * Returns an empty array if the collection is empty or null.
+ */
+function getTopVoted(pollAnswers) {
+  if (!pollAnswers || pollAnswers.size === 0) {
+    return [];
+  }
+
+  let maxVoteCount = 0;
+  const topVoted = [];
+
+  for (const pollAnswer of pollAnswers.values()) {
+    if (pollAnswer.voteCount > maxVoteCount) {
+      maxVoteCount = pollAnswer.voteCount;
+      topVoted.length = 0;
+      topVoted.push(parseInt(pollAnswer.text, 10));
+    } else if (
+      pollAnswer.voteCount !== 0 &&
+      pollAnswer.voteCount === maxVoteCount
+    ) {
+      topVoted.push(parseInt(pollAnswer.text, 10));
+    }
+  }
+
+  return topVoted.length > 0 ? topVoted : [maxVoteCount];
```

Comment:
Try to modify the function so always returns the top pollAnswer 

now it return one answer OR an array, is confusing